Library/Formula/strongswan.rb

```diff
@@ -15,6 +15,15 @@ class Strongswan < Formula
 
   depends_on 'openssl'
   depends_on 'curl' => :optional
+  depends_on 'autoconf' => :build
```

Comment:
If these aren't required for the stable build you shouldn't put them in the core formula, otherwise everyone will require them regardless of whether they build head or not. Should be:

```ruby
head do
  url "https://git.strongswan.org/strongswan.git"

  depends_on "autoconf" => :build
  depends_on "automake" => :build
```

And so on.